src/Components/LoginForm.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { Container, Input, Icon, Button } from 'semantic-ui-react'
+
+const LoginForm = (props) => {
+  return (
+    <>
+      <Container id="login-form">
+        <Input iconPosition='left' placeholder="Email" id="email-input" name="email" onChange={props.inputChangeHandlerProps}>
+          <Icon name='at' />
+          <input />
+        </Input>
+        <Input id="password-input" placeholder="Password" name="password" onChange={props.inputChangeHandlerProps}>
+          <input />
```

Comment:
Remove redundant closing tag.

Reply:
No we need that tag because otherwise there is no input field.

Reply:
Same as above.

Reply:
Why would you need a closing tag that has no corresponding opening tag?
```
        <**Input** iconPosition='left' placeholder="Email" id="email-input" name="email" onChange={props.inputChangeHandlerProps}>
          <Icon name='at' />
          <input />
        **</Input>**
        **<Input** id="password-input" placeholder="Password" name="password" onChange={props.inputChangeHandlerProps}>
          <input />
        **</Input>**
```